frontend/src/component/feature/FeatureStrategy/FeatureStrategySegment/RecentlyUsedSegments/RecentlyUsedSegments.tsx

```diff
@@ -54,12 +54,10 @@ export const RecentlyUsedSegments = ({
             <StyledHeader>Recently used segments</StyledHeader>
             <StyledSegmentsContainer>
                 {segmentObjects.map((segment) => (
```

Comment:
Should we do some kind of filtering here to hide segments that already exist on that strategy? but ... If we only show recent segments when you have no segments, then that doesn't matter.